esmvaltool/cmorizers/obs/cmorize_obs_duveiller2018.py

```diff
@@ -0,0 +1,152 @@
+"""ESMValTool CMORizer for Duveiller2018 data.
+
+Tier
+   Tier 2: other freely-available dataset.
+
+Source
+   https://www.nature.com/articles/sdata201814
+
+Last access
+   20190430
+
+Download and processing instructions
+   - Download the dataset albedo_IGBPgen.nc and save in the right directory
+     according to ESMValTool practices.
+   - Complete the CMOR-config specifications (see instructions in the file
+     itself)
+   - Run cmorize_obs
+
+Modification history
+   20190701-A_crez_ba: submitted PR
```

Comment:
```suggestion
```

Reply:
Why take this out? I think it is useful information, basically the stage between submitting the PR and the closing of the PR is a phase in which the code is improved thanks to automated tests and `peer reviewing`. :)

Reply:
I think it is not necessary to go in such a detail in this section.
We just document the major changes in the script here.